joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/dataflow/IntraMethodDataflowTests.scala

```diff
@@ -29,4 +29,35 @@ class IntraMethodDataflowTests extends PhpCode2CpgFixture(runOssDataflow = true)
 
     flows.size shouldBe 1
   }
+
+  "flow from array unpacking should be found" in {
+    val cpg = code("""<?php
+        |[$a1, $a2] = $arr1;
+        |echo $a1;
+        |[[$b1, $b2]] = $arr2;
+        |echo $b1;
+        |list(list($c1, $c2)) = $arr3;
+        |echo $c1;
+        |""".stripMargin)
+    {
+      val source = cpg.identifier("arr1")
+      val sink   = cpg.call("echo")
```

Comment:
The "more precise" sink for what you want looks like this. Tagging a call is generally seen as tagging its return.
```suggestion
      val sink   = cpg.call("echo").argument(1)
```